eox_tenant/tenant_aware_functions/enrollments.py

```diff
@@ -10,6 +13,10 @@ def filter_enrollments(enrollments):
     do not belong to the current microsite
     """
 
+    if getattr(settings, "EOX_TENANT_AWARE_FUNCTION_FILTER_ENROLLMENTS", False):
```

Comment:
We can make this be one the boolean checks. The second would be the `not get_microsite().is_request_in_microsite()` and we can use the function already on L21

Reply:
Also, the setting could be way clearer: `EOX_TENANT_SKIP_FILTER_FOR_TESTS`